typos_correction/16577a2c-7f17-4a6f-a759-92f3a00cf339.md

```diff
@@ -0,0 +1,31 @@
+# typos_correction
+
+Model that suggests fixes to correct typos.
```

Comment:
Please change this tautology.

E.g. `Model to correct typos in source code identifiers.`